Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -185,7 +189,7 @@ WarpX::AddSpaceChargeFieldLabFrame ()
 #ifndef AMREX_USE_EB
     computeE( Efield_fp, phi_fp, beta );
 #else
-    if (warpx_py_poissonsolver) computeE( Efield_fp, phi_fp, beta );
+    if ( external_solver ) computeE( Efield_fp, phi_fp, beta );
```

Comment:
Looking at this: what happens if we have EB compiled in but no external Poisson solver? looks like we skip the `computeE` step then. (Unrelated to this PR, of course.)

Reply:
When compiled with EB support the E field is calculated using the AMReX functionality, `mlmg.getGradSolution`, from inside the `WarpX::computePhi` function. Without EB support the function `WarpX::computeE` calculates the E-field. It would be good to switch over the last remaining case, XZ or XYZ geometry without EB support, to also use the `mlmg.getGradSolution` approach.

Reply:
Thanks, got ya! Do you like to open an issue for that to track this?

Reply:
Will do!